common/convert/convert.go

```diff
@@ -59,6 +60,16 @@ func StringPtr(v string) *string {
 	return &v
 }
 
+// TimePtr makes a copy and returns the pointer to a time.Time.
+func TimePtr(v time.Time) *time.Time {
+	return &v
+}
+
+// DurationPtr makes a copy and returns the pointer to a time.Duration.
+func DurationPtr(v time.Duration) *time.Duration {
+	return &v
+}
```

Comment:
We have these in `timestamp` package. Bad name, need to be refactored but please use it for now. I will change it later.